### PR TITLE
ROX-19183: Add ImageManagementTest to the parallel set

### PR DESCRIPTION
**Pull request by gavin-stackrox**
## Description

This PR adds `ImageManagementTest.groovy` to the set of tests that run concurrently. This is achieved with a standard pattern: clone and scope policies to a test namespace and create deployments in that same namespace.

## Checklist
- [x] Investigated and inspected CI test results

## Testing Performed

50x runs:
- testParallel - [out43.txt](https://github.com/stackrox/stackrox/files/12412001/out43.txt)
- testParallelBAT - [out42.txt](https://github.com/stackrox/stackrox/files/12411998/out42.txt)

### CODE REVIEW

File: qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy [hunk 1]
```diff
@@ -135,6 +135,7 @@ class ProcessBaselinesTest extends BaseSpecification {
 
     def cleanupSpec() {
         PolicyService.deletePolicy(unauthorizedProcessExecution.getId())
+        orchestrator.deleteNamespace(TEST_NAMESPACE)
```
Comment (gavin-stackrox) [inline qa-tests-backend/src/test/groovy/ProcessBaselinesTest.groovy:138]: This missing NS delete showed up as a backtrace while debugging this PR. Strictly speaking it should have been added when `ProcessBaselinesTest` was parallelized (#7474) and surfaces an issue that `compareResourcesAtRunEnd()` does not fail the test now that it is executed in the shutdown handler.

**Review by theencee — APPROVED** [review]

**Pull request merged**